cgmanifest.json

```diff
@@ -4620,8 +4620,28 @@
         "type": "other",
         "other": {
           "name": "golang",
-          "version": "1.21.9",
-          "downloadUrl": "https://github.com/microsoft/go/releases/download/v1.21.9-1/go.20240403.5.src.tar.gz"
+          "version": "1.19.12",
```

Comment:
No, the intent here is only to build+publish a single version of golang. I'm honestly not all that familiar with this file and what should be included--I have these extra versions listed because the spec uses them to bootstrap the final version. I guess the fact that Go 1.4 *isn't* in this file should have tipped me off. 😄 

Something that confuses me about this file is why there are entries for `https://golang.org/dl/go1.18.8.src.tar.gz` and `https://golang.org/dl/go1.20.10.src.tar.gz`, neither of which show up in the repo. Cleaned those up too.